src/components/Tweets/Tweet.js

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes } from 'react';
+import cx from 'classnames';
+
+import styles from './Tweet.scss';
+
+const Tweets = (props) => {
+  return (
+      <div className={cx(styles.block)}>
+        <div className={cx(styles.image)}>
+          <img width="48" height="48" src={props.user.profile_background_image_url}/>
+        </div>
+        <div className={cx(styles.tweet)}>
+          <li><span className={cx(styles.fullName)}>{props.user.name}</span> - <span className={cx(styles.screenName)}>@{props.user.screen_name}</span></li>
```

Comment:
Thanks :)

You also don't need the cx() calls here. cx() lets you add multiple classes at once but if you're just using one class, it's fine to write `className={styles.fullName}` 

Can you split this line onto multiple lines as well? When eslint kicks in again, the 100 character line limit will kick in too.